### PR TITLE
Navbar

### PR DESCRIPTION
**Pull request by kbusemeyer**

### CODE REVIEW

File: templates/firstbyte/base.html [hunk 1]
```diff
@@ -152,7 +119,7 @@ <h6 class="footer-nav-header mb-0">ADDITIONAL LINKS</h6>
                         <span class="text-muted px-2">|</span>
                         <span class="text-muted"><a href="https://github.com/lukesanantonio/firstbyte" target="blank">View project on Github</a></span>
                     </div>
-                </div>
+                </div>
```
Comment (lukesanantonio) [inline templates/firstbyte/base.html]: I know it's nitpicky :smile: but could you trim whitespace if possible? (Usually your editor will have a setting)

File: templates/firstbyte/base.html [hunk 2]
```diff
@@ -21,7 +21,7 @@
     {% block ga %}{% if GA_TRACKING_ID %}{% include 'firstbyte/ga.html' %}{% endif %}{% endblock %}
 </head>
 <body class="d-flex flex-column">
-    <nav class="sticky-top navbar navbar-expand-lg navbar-dark bg-dark py-3">
+    <nav class="sticky-top navbar navbar-expand-lg bg-green py-3">
```
Comment (lukesanantonio) [inline templates/firstbyte/base.html]: Okay, I think I was able to get the desired output by reverting the changes to bootstrap.css while using `navbar-light bg-green`. I believe `navbar-dark` just changes default text color whereas `bg-green` sets the background. Please let me know if I'm misunderstanding the issue!

File: firstbyte/static/firstbyte/style/main.css [hunk 3]
```diff
@@ -1,143 +1,158 @@
 /* Fonts */
 
 body {
-    font-family: 'Open Sans', sans-serif;
+  font-family: "Open Sans", sans-serif;
```
Comment (lukesanantonio) [inline firstbyte/static/firstbyte/style/main.css]: I totally support sticking to automatically formatting our code (share your settings?) but I think we should keep it to it's own PR. I'd be happy to just reset the formatting myself and commit here, is that alright with you?

File: firstbyte/static/firstbyte/style/main.css [hunk 4]
```diff
@@ -1,143 +1,158 @@
...
-    font-size: 1rem;
-    font-weight: bold;
-    padding-top: 1.1rem;
-    padding-bottom: .2rem;
+  font-size: 0.75rem;
+  font-weight: bold;
+  padding-top: 1.1rem;
+  padding-bottom: 0.2rem;
+}
+.footer-license {
```
Comment (lukesanantonio) [inline firstbyte/static/firstbyte/style/main.css]: All the changes to the CSS look good to me :+1:

File: firstbyte/static/firstbyte/style/main.css [hunk 5]
```diff
@@ -1,143 +1,158 @@
...
-    font-size: 1em;
+  color: white !important;
+  font-weight: 500;
+  font-size: 0.875em;
+  margin-bottom: 0px;
 }
 
 .nav-link:hover {
-    color: rgba(255, 255, 255, 1) !important;
+  color: rgba(255, 255, 255, 1) !important;
```
Comment (lukesanantonio) [inline firstbyte/static/firstbyte/style/main.css]: If we set the color of .nav-link to white, we don't need to handle hovers I think.

File: templates/firstbyte/base.html [hunk 6]
```diff
@@ -32,6 +32,7 @@
 
         <div id="navbar-links" class="collapse navbar-collapse">
             <div class="navbar-nav">
+                <p class="nav-item nav-link mx-2">FIRSTBYTE</p>
```
Comment (lukesanantonio) [inline templates/firstbyte/base.html]: Should we remove the logo in this PR, what do you think?

File: firstbyte/static/firstbyte/style/main.css (hunk 3 above)
Comment (kbusemeyer) [reply firstbyte/static/firstbyte/style/main.css]: Yes that's fine, I'm just using the prettier formatter I think and didn't realize that it had changed that - I can probably adjust my settings to match too though.

File: templates/firstbyte/base.html (hunk 6 above)
Comment (kbusemeyer) [reply templates/firstbyte/base.html]: We probably should, with the color change it's not very noticeable anyways.
